elements/lisk-chain/src/block.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+import { codec } from '@liskhq/lisk-codec';
+import { BlockHeader } from './block_header';
+import { blockSchema } from './schema';
+import { Transaction } from './transaction';
+
+interface BlockAttrs {
+	header: Buffer;
+	payload: Buffer[];
+}
+
+export class Block {
```

Comment:
The issue says it should have 2 more methods,

```js
public get header() BlockHeader;
public get payload() []Transaction;
```

Reply:
Not required to to have getters as the `header` and `payload` both are public readonly properties.